hack/rename-project

```diff
@@ -16,8 +16,8 @@
 
 project_name=$1
 
-sed -i '' "s/gardener/$project_name/g" pkg/sampleprovider/driver.go
-sed -i '' "s/gardener/$project_name/g" pkg/sampleprovider/machineserver.go
-sed -i '' "s/gardener/$project_name/g" app/controller/main.go
+sed -i "s/gardener/$project_name/g" pkg/sampleprovider/driver.go
```

Comment:
This works with both GNU and BSD versions of sed. 

sed -i '' -e "s/gardener/$project_name/g" pkg/sampleprovider/driver.go

I am making this change in the upcoming PR here - https://github.com/gardener/machine-controller-manager-provider-sampleprovider/pull/4/commits/21ce1759065b743fda197bc923a844ea463a0fa4. Let me know if this command works on your version of sed. if it does, i could make these changes in the upcoming PR maybe?

Reply:
I could make this work locally on my side. So you can go ahead an close this PR if you are already working on a fix.